rtb/verbali/interni/VI_2024-11-10_ver1.0.0.typ

```diff
@@ -0,0 +1,75 @@
+#import "/templates/template.typ": *
+
+#show: content=>verbale(
+  data: "10 novembre 2024", 
+  responsabile: "Riccardo Milan", 
+  redattori: ("Andrea Precoma",), 
+  titolo: "Terza riunione", 
+  verificatori: (
+    "Andrea Perozzo",
+    "Davide Picello",
+    "Klaudio Merja"
+  ),
+  riassunto: "Discussione principale riguardo alla struttura del backlog, alla suddivisione delle repository e al sistema di pubblicazione della documentazione. Miglioramento del Way of Working.",
+  versioni: (
+    "1.0.0", 
+    [10/11/2024], 
+    "Andrea Precoma", 
+    "Andrea Perozzo\nDavide Picello\nKlaudio Merja",
+    [- Stesura del verbale],
+  ),
+  content: content
+)
+
+#infoRiunione(
+  data: "10/11/2024", durata: "2 ore e 10 minuti", luogo: "Chiamata Discord", ora: "15:15", partecipantiEsterni: (),
+)
+
+= Ordine del giorno
+== Riunione esterna con l'azienda proponente
+In seguito alla vittoria della candidatura per il capitolato C4 abbiamo provveduto a pianificare una e-mail per lunedì 11 novembre per concordare un incontro con l'azienda proponente. Abbiamo chiesto disponibilità dopo il "Diario di Bordo" e la lezione rovesciata in modo da avere qualche chiarimento anche dal professor Vardanega e avere più materiale sul quale discutere.
+
+== Sito per la documentazione
+Dopo alcune proposte stilistiche riguardo al sito dove pubblicare la documentazione abbiamo deciso di aggiornarci alla prossima riunione. Oltre a sistemare l'aspetto del sito bisogna aggiungere la logica che permette di collegare i link del sito ai documenti in pdf. Abbiamo quindi creato la repository *sweatunipd.github.io* che conterrà i file Python che aggiorneranno i link del sito in modo che aprano i documenti desiderati.
```

Comment:
La nuova repo non contiene SOLO gli script in Python, ma è invece il sito principale del gruppo dove sono contenute sia le info dello stesso, sia la documentazione. Da chiarire invece se a sto punto togliere GitHub Directory Listing

Reply:
Quindi a questo punto la repo dovrebbe contenere anche i pdf dei documenti che compila la action?

Reply:
Nope... se li va a prendere e basta... i pdf sono già generati e buttati su github pages... per quello dico: bisogna capire se tenere o meno github directory listing @ErPreco

Reply:
Mi spiego meglio: il sito deve essere un update del design che abbiamo già implementato (ovvero il solito Directory Listing) e deve poter recuperare i file senza farlo fare a noi a mano ogni volta. Bisogna provare e vedere se funziona.

Reply:
Capito, quindi per ora scrivo che la repo contiene i file relativi alla struttura del sito (html, css, js) e che prossimamente miglioreremo il da farsi. Confermi @klamerja ?

Reply:
Sì. Per ora sì